tpot/base.py

```diff
@@ -519,7 +523,31 @@ def _set_param_recursive(self, pipeline_steps, parameter, value):
                 if hasattr(obj, parameter):
                     setattr(obj, parameter, value)
 
-    def _evaluate_individual(self, individual, features, classes):
+    def _set_sample_weight(self, pipeline_steps, sample_weight):
```

Comment:
We shouldn't need a new function for this; we can use the existing `_set_param_recursive` function.